src/dbt/kipptaf/models/alchemer/base/base_alchemer__survey_results.sql

```diff
@@ -7,8 +7,8 @@ select
     sr.session_id as response_session_id,
     sr.contact_id as response_contact_id,
     sr.status as response_status,
-    sr.date_started as response_date_started,
-    sr.date_submitted as response_date_submitted,
+    timestamp(sr.date_started) as response_date_started,
+    timestamp(sr.date_submitted) as response_date_submitted,
```

Comment:
can you make these new columns (i.e. `response_date_started_timestamp`) and preserve the originals?